src/stacktrace.rs

```diff
@@ -0,0 +1,247 @@
+extern crate lazy_static;
+
+use crate::error::*;
+use gdb_command::stacktrace::*;
```

Comment:
Create a type alias Stacktrace as gdb_command::stacktrace and replace all type usages in casr

Reply:
done